### PR TITLE
Issue37: global_memory

### PR DESCRIPTION
**Pull request by shamith2**
#37

### CODE REVIEW

File: cap_test.py [hunk 1]
```diff
@@ -57,4 +57,3 @@
     del gym.envs.registry.env_specs['cap-v0']
 
     print("CtF environment Closed")
-
```
Comment (skim0119) [inline cap_test.py]: Remove unnecessary commit

File: gym_cap/envs/cap_env.py [hunk 2]
```diff
@@ -385,7 +388,35 @@ def seed(self, seed=None):
         """
         self.np_random, seed = seeding.np_random(seed)
         return [seed]
-
+        
+    def global_memory(self):
```
Comment (skim0119) [inline gym_cap/envs/cap_env.py]: Change method name to 'update global memory'

**Review by skim0119 — COMMENTED** [review]
The code is not running, probably due to the same reason #40 #12.

I will push this and individual memory setting once the result is checked with rendered image.

File: gym_cap/envs/cap_env.py (hunk 2 above)
Comment (shamith2) [reply gym_cap/envs/cap_env.py]: changed accordingly

File: cap_test.py (hunk 1 above)
Comment (shamith2) [reply cap_test.py]: removed

**Pull request closed without merging**